### PR TITLE
initial commit and some content

### PR DESCRIPTION
**Pull request by kaleem99**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/topics/web-frontend/react/redux-architecture/_index.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+  hard:
+  - topics/redux-intro/
+  soft: []
+ready: true
 title: React + Redux architectural guidelines
----
\ No newline at end of file
+---
+
+Redux is a very powerful state management library used in JavaScript apps to mange the data of your application and was originally designed for react. Redux helps you write applications that behave consistently and can run in different environments.
```
Comment (FaithMo) [inline content/topics/web-frontend/react/redux-architecture/_index.md]: Looks good, just fix the sentence structure. e.g
"..in Javascript apps to manage the data of an application, and was originally designed for React..."
and
"Redux helps you write applications that behave consistently by facilitating and sharing data across the components of an application..."

**Review by FaithMo — CHANGES_REQUESTED** [review]

**Review by FaithMo — APPROVED** [review]

**Review by babalwa-mbolekwa — APPROVED** [review]

**Review by ry-oc — APPROVED** [review]

File: content/topics/web-frontend/react/redux-architecture/_index.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+  soft: []
+ready: true
 title: React + Redux architectural guidelines
----
\ No newline at end of file
+---
+
+Redux is a very powerful state management library used in JavaScript apps to manage the data of your application, and was originally designed for React. Redux helps you write applications that behave consistently by facilitating and sharing data across the components of an application.
+
+### Fundamental principles
```
Comment (sheenarbw) [inline content/topics/web-frontend/react/redux-architecture/_index.md]: This should be a level 2 heading.

Comment (sheenarbw) [reply content/topics/web-frontend/react/redux-architecture/_index.md]: And I dont think you are actually covering any architectural guidelines here...

If you have content that should be added to topics/redux-intro/ then it should go there, not here. 

Eg getting started with redux is definately introductiory material. It has little to do with scalable architectures. Same can be said for "thinking in redux"

**Review by sheenarbw — CHANGES_REQUESTED** [review]
This PR adds content to the redux architecture topiuc that has very little to do with redux architecture.

It also doesn't follow the style gude because heading levels are skipped.

**Comment by ry-oc** [comment]
No longer relevant

**Pull request closed without merging**